From: Ansgar Burchardt <ansgar@debian.org>
Date: Sun, 13 Oct 2013 17:57:22 +0200
Subject: disable searchengine in doxygen HTML output

Origin: vendor
Forwarded: not-needed

Disable the searchengine in doxygen's HTML output to avoid an embedded copy of
jquery.js to be pulled in.

Reference: http://bugs.debian.org/622147
---
 doc/dox/Doxyfile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/dox/Doxyfile.in b/doc/dox/Doxyfile.in
index ddbf353..d29cb6e 100644
--- a/doc/dox/Doxyfile.in
+++ b/doc/dox/Doxyfile.in
@@ -164,7 +164,7 @@ TREEVIEW_WIDTH         = 190
 EXT_LINKS_IN_WINDOW    = NO
 FORMULA_FONTSIZE       = 10
 FORMULA_TRANSPARENT    = YES
-SEARCHENGINE           = YES
+SEARCHENGINE           = NO
 SERVER_BASED_SEARCH    = NO
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
